global-api/importer/ghgrp_epa/lat_lon_to_locode.py

```diff
@@ -0,0 +1,121 @@
+import re
+from shapely.geometry import Point
+from shapely.wkt import loads
+from sqlalchemy import text
+
+
+def point_to_lat_lon(point):
```

Comment:
Don't we already have this for the other data sources? Would be good to prevent duplication here.